stdlib/public/core/UnsafeRawPointer.swift.gyb

```diff
@@ -530,6 +538,17 @@ public struct Unsafe${Mutable}RawPointer : Strideable, Hashable, _Pointer {
   }
 
 %  if mutable:
+
+  @available(swift, obsoleted: 5.0.0, renamed: "initializeMemory(as:repeating:count:)")
+  @discardableResult
+  public func initializeMemory<T>(
+    as type: T.Type, at offset: Int = 0, count: Int = 1, to repeatedValue: T
+  ) -> UnsafeMutablePointer<T> { 
+    _debugPrecondition(offset == 0,
```

Comment:
This new precondition will break the behavior of existing code that uses a nonzero `at` parameter.

Reply:
the behavior would already be broken anyway since the new implementation doesn’t take an offset parameter, so it wouldn’t get forwarded from the compatibility shim. The precondition just warns users about this.

Reply:
nvm i just used `(self + offset * MemoryLayout<T>.stride)` in the shim body

Reply:
Great solution! 👍